data/saas/config/rollbar_config.yml

```diff
@@ -0,0 +1,104 @@
+saas_config:
+  fides_key: <instance_fides_key>
+  name: Rollbar SaaS Config
+  type: rollbar
+  description: A sample schema representing the Rollbar connector for Fidesops
+  version: 0.0.1
+
+  connector_params:
+    - name: domain
+    - name: read_access_token
+    - name: write_access_token
+
+  client_config:
+    protocol: https
+    host: <domain>
+
+  test_request:
+    method: GET
+    path: /api/1/status/ping
+
+  endpoints:
+    - name: projects
+      requests:
+        read:
+          method: GET
+          path: /api/1/projects
+          headers:
+            - name: X-Rollbar-Access-Token
+              value: <read_access_token>
+          param_values:
+            - name: read_access_token
+              connector_param: read_access_token
+            - name: placeholder
+              identity: email
+          data_path: result
+    - name: project_access_token
+      requests:
+        read:
+          method: GET
+          path: /api/1/project/<project_id>/access_tokens
+          headers:
+            - name: X-Rollbar-Access-Token
+              value: <read_access_token>
+          param_values:
+            - name: read_access_token
+              connector_param: read_access_token
+            - name: project_id
+              references:
+                - dataset: <instance_fides_key>
+                  field: projects.id
+                  direction: from
+          postprocessors:
+            - strategy: filter
+              configuration:
+                field: name
+                value: read
+          data_path: result
+    - name: instances
+      requests:
+        read:
+          method: GET
+          path: /api/1/instances
+          headers:
+            - name: X-Rollbar-Access-Token
+              value: <project_read_token>
+          query_params:
+            - name: page
+              value: 1
+          param_values:
+            - name: project_read_token
+              references:
+                - dataset: <instance_fides_key>
+                  field: project_access_token.access_token
+                  direction: from
+          pagination:
+            strategy: offset
+            configuration:
+              incremental_param: page
+              increment_by: 1
+              limit: 20
```

Comment:
Create a `page_limit` connector_param and use it here to let the user configure how many pages to paginate through.

Reply:
Logically, the user wants to delete the records for a given identity(email) even if it occurs on the last page.
Do, you still think we should give access to configure the page limit?